tests/integrationv2/constants.py

```diff
@@ -1,2 +1,5 @@
 TEST_CERT_DIRECTORY="../pems/"
 TEST_SNI_CERT_DIRECTORY="../pems/sni/"
+
+TRUST_STORE_BUNDLE=TEST_CERT_DIRECTORY + 'trust-store/ca-bundle.crt'
+TRUST_STORE_TRUSTED_BUNDLE=TEST_CERT_DIRECTORY + 'trust-store/ca-bundle.trust.crt'
```

Comment:
Q: Why are we copying the CA bundle to a new location? What is the problem faced in pointing to the current location `tests/integration/trust-store`?

Reply:
That location will go away once we deprecate the existing framework. I think the bundle really belongs under the pems directory anyway, and not in the original integration location.

This just lets us deprecate the old framework without having to update any file locations.